docs/changelog.rst

```diff
@@ -77,6 +77,9 @@ Core API
 - Add :meth:`mopidy.core.PlaylistsController.get_uri_schemes`. (PR:
   :issue:`1362`)
 
+- Persist state between runs. The amount of data to persist can be
+  controlled by config value :confval:`core/restore_state`
```

Comment:
This must be moved to the 2.1.0 changelog.

Also, the wording should be changed as it is not yet possible to control what to persist through the config value.